src/hermes/commands/workflow.py

```diff
@@ -112,12 +112,22 @@ def deposit():
     Deposit processed (and curated) metadata
     """
     click.echo("Metadata deposition")
+    _log = logging.getLogger("cli.deposit")
```

Comment:
Does this work without the logger being configured? (I guess because the `cli` is defined... but not sure here)

Reply:
I don't think the name for the logger can be entirely made up. The docs say a hierarchical format is typical but they make no requirements. See [here](https://docs.python.org/3/library/logging.html#logging.getLogger).

Reply:
Yes, the name itself is in fact not constrained. However, in [src/hermes/config.py](config.py) I had to provide some basic configuration for the loggers. But I think the `cli` logger being defined there is sufficient for your code to actually produce log output. So should be fine here .:)

Reply:
Oh I see. Yes, it does work. You can verify this by deleting you `.hermes` directory and then running `hermes deposit`. This should print "You must run the process command before deposit"

Reply:
And this line is just a copy of these two:

https://github.com/hermes-hmc/workflow/blob/820c813b607d3ab0ad1ccb070c1611482cdadfd8/src/hermes/commands/workflow.py#L25

https://github.com/hermes-hmc/workflow/blob/820c813b607d3ab0ad1ccb070c1611482cdadfd8/src/hermes/commands/workflow.py#L57